src/ProofOfConcepts/connectors/connectors.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import React from "react";
+
+import { DataListSelectableRows } from "./connectors";
+
+export default {
+  component: DataListSelectableRows,
+  args: {},
+} as ComponentMeta<typeof DataListSelectableRows>;
+
+const Template: ComponentStory<typeof DataListSelectableRows> = (args) => (
+  <DataListSelectableRows {...args} />
+);
+
+export const Story = Template.bind({});
```

Comment:
It's worth getting into the habit of avoiding generic names, often you'll find you'll want to add a second story and then you have to go back and rename this one :smile: